### ISSUE TITLE
Allow to categorize by license type

### ISSUE THREAD

**Issue by dpsanders** [openjournals/joss#597]
Perhaps the keywords could make the license type explicit, or it should be a separate field that can be sorted / filtered on.

**Comment by dpsanders** [comment]
I was referring to the software license for the code.

**Comment by arfon** [comment]
Good idea. Unfortunately we don't currently have this captured in the metadata anywhere for the submissions but I agree this would be a neat thing to be able to search on.
